### DISCUSSION TITLE
[Support]: 0.15.* Single event retention past retain: setting

### DISCUSSION THREAD

**Discussion by csreynolds** [blakeblackshear/frigate#15537] [General Support] [answered]
### Describe the problem you are having

Pre 0.15 there was an event star system that would retain an event past the retention setting. I don't see anything that does that in the UI currently in 0.15. 

Am i just missing something obvious?

### Version

 0.15.0-869fa26

### Frigate config file

```yaml
N/A
```

### Relevant Frigate log output

```shell
N/A
```

### Relevant go2rtc log output

```shell
N/A
```

### FFprobe output from your camera

```shell
N/A
```

### Install method

HassOS Addon

### docker-compose file or Docker CLI command

```yaml
N/A
```

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

N/A

**Comment by hawkeye217** [comment, marked as answer]
No, this was removed in 0.14 in favor of exports. See the [FAQ](https://github.com/blakeblackshear/frigate/discussions/12724), Usage question 11.

**Reply by csreynolds** [reply]
Oooohhh I see. 

I always gravitate to /explore instead of /review. 

Looks like the only place to export is from /review right now.

**Reply by hawkeye217** [reply]
Correct, you can export review items. You can also export from the History view. 

Additionally, you can download individual tracked object video clips from the Explore page. This just doesn't save it in the Exports page of the UI.
